build.gradle

```diff
@@ -16,11 +20,13 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.3'
+        classpath 'com.android.tools.build:gradle:4.2.0'
         classpath 'com.google.gms:google-services:4.3.5'
-        classpath 'com.google.firebase:firebase-crashlytics-gradle:2.5.1' // Crashlytics
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath 'com.google.firebase:firebase-crashlytics-gradle:2.5.1' // Crashlytics
         classpath "com.google.dagger:hilt-android-gradle-plugin:$hiltVersion"
+        classpath "androidx.navigation:navigation-safe-args-gradle-plugin:$navigationVersion"
+        classpath "com.google.android.libraries.mapsplatform.secrets-gradle-plugin:secrets-gradle-plugin:1.3.0"
```

Comment:
I wonder if there is something like this that could be used to project other secrets unrelated to map keys.

Reply:
It seems like this plugin actually **can** be used for any other secrets in the `local.properties`: https://github.com/google/secrets-gradle-plugin

Reply:
Yes, I will make a Story for this.